examples/server/tests/unit/test_completion.py

```diff
@@ -143,13 +143,15 @@ def test_consistent_result_same_seed(n_slots: int):
 def test_different_result_different_seed(n_slots: int):
     global server
     server.n_slots = n_slots
+    server.n_predict = -1
```

Comment:
I don't get why it is needed here. The default value of `n_predict` is already `-1`, you should not change a test case that is unrelated the the currently change, that's the whole I idea of having tests: to make sure that your change does not break exiting use cases.

Reply:
There is a bit of a readability and maintainability issue in this file in that the server object is global, and as a test fixture is resused across tests, even though every test changes its parameters.